.github/dependabot.yml

```diff
@@ -26,6 +34,10 @@ updates:
         update-types:
         - "version-update:semver-major"
         - "version-update:semver-minor"
+    groups:
+      k8s:
+        patterns:
+        - "k8s.io/kubernetes"
```

Comment:
this is a single dependency. we can do k8s.io but we'll have to exclude cri. looks like we can have exclusions: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file